iwyu_use_flags.h

```diff
@@ -17,7 +17,7 @@ typedef unsigned UseFlags;
 
 const UseFlags UF_None = 0;
 const UseFlags UF_InCxxMethodBody = 1;       // use is inside a C++ method body
-const UseFlags UF_DefinitionUse = 2;         // use is itself a definition
+const UseFlags UF_Exact = 2;                 // don't remap to another decl
```

Comment:
Hehe, I know I say this every time anybody touches these flags, which indicates that maybe I'm doing it backwards :slightly_smiling_face::

The flags are intended to say something about the context in which the use occurs, not about what you want to happen in use reporting.

I'm not sure it's possible to generalize both definition-is-use-of-earlier-declaration and decl-has-a-prior-decl-with-default-values-for-later-params into the same concept. Might be better to add a new flag and check both.